python/drned_xmnr/op/setup_op.py

```diff
@@ -30,16 +35,16 @@ class DevcliLogMatch(object):
         r'|(?P<timeout>device communication failure: .*Timeout.*)'
         r'|(?P<authfailed>failed to authenticate)'
         r')$')
-    matchrx = re.compile(matchexpr)
+    matchrx: Pattern[str] = re.compile(matchexpr)
```

Comment:
I think you are overdoing this, mypy is (should be) able to derive the type.

Reply:
will remove